src/config.js

```diff
@@ -46,12 +55,12 @@ const conf = convict({
   db: {
     host: {
       doc: 'Hostname where PostgreSQL is running',
-      format: 'url',
+      format: '*',
       default: 'localhost'
     },
     port: {
       doc: 'Port where PostgreSQL is running',
-      format: 'port',
+      format: '*',
```

Comment:
I think it's probably better to do something in a separate patch; possibly with a custom verifier. I'll revert this from this patch for now.